homeassistant/components/govee_ble/__init__.py

```diff
@@ -14,37 +14,32 @@
 from homeassistant.const import Platform
 from homeassistant.core import HomeAssistant
 
-from .const import DOMAIN
-
 PLATFORMS: list[Platform] = [Platform.SENSOR]
 
 _LOGGER = logging.getLogger(__name__)
 
+GoveeBLEConfigEntry = ConfigEntry[PassiveBluetoothProcessorCoordinator]
```

Comment:
```suggestion
GoveeBLEConfigEntry = ConfigEntry[PassiveBluetoothProcessorCoordinator[SensorUpdate]]
```
Should the generic type be `SensorUpdate` here (from `bthome_ble`)?

Reply:
616dae73ab15a481de8ba8863ec7d7c81d079fb7